l10n_it_fatturapa_pec/models/mail_thread.py

```diff
@@ -15,9 +15,13 @@
 
 _logger = logging.getLogger(__name__)
 
-FATTURAPA_IN_REGEX = "^[A-Z]{2}[a-zA-Z0-9]{11,16}_[a-zA-Z0-9]{,5}.(xml|zip)"
-RESPONSE_MAIL_REGEX = '[A-Z]{2}[a-zA-Z0-9]{11,16}_[a-zA-Z0-9]{,5}_MT_' \
-                      '[a-zA-Z0-9]{,3}'
+FATTURAPA_IN_REGEX = '^(IT[a-zA-Z0-9]{11,16}|'\
+                     '(?!([A-Z]{2})?IT)[a-zA-Z0-9]{2,28})'\
+                     '_[a-zA-Z0-9]{,5}.(xml|XML|zip|ZIP)'
```

Comment:
Questo però fa match anche con `SM_12345.xml` e penso non dovrebbe

Reply:
Vero, e come lo correggi? Tra parentesi, JS e PHP non matcha

Reply:
Se non si riesce a fare di meglio con regex, direi che comunque vale la pena intanto tenere questa soluzione, rispetto a quella attuale che scarta fatture

Reply:
forse la soluzione è {4,28}

Reply:
@sergiocorato forse sì, ma non mi è chiara la parte `(?!([A-Z]{2})?IT)`

Reply:
@eLBati Per accettare tutte le stringhe di 2 caratteri eccetto IT, non ti torna?

Reply:
mmm {4,30} mi sa